.github/workflows/rust.yml

```diff
@@ -37,46 +37,29 @@ jobs:
       - name: Cache Rust stuff
         uses: Swatinem/rust-cache@v1
 
-      - name: Setup OCaml (because of ocaml-gen)
-        run: |
-          sudo apt update
-          sudo apt install -y ocaml
```

Comment:
I don't think CI is going to work without OCaml (due to ocaml-rs that requires it)

Reply:
CI is happy, we don't have anything that uses OCaml directly. This was only here for the ocaml-gen tests, which are now in another repo.

Reply:
this is not tested in CI until it lands. I'll approve it but I'm pretty sure it's going to break CI :D as it was ocaml-rs that required it, not ocaml-gen (and we build with all features)